wp-content/themes/pub/wporg-learn-2020/template-parts/component-author.php

```diff
@@ -1,9 +1,27 @@
+<?php
+/**
+ * Template part for display author information
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package WPBBP
+ */
+
+$display_name = get_the_author_meta( 'display_name' );
+$nickname = get_the_author_meta( 'nickname' );
+$user_url = get_the_author_meta( 'url' );
+$url = get_avatar_url( get_the_author_meta( 'ID' ) );
+
+?>
+
 <div class="workshop-author">
 	<div>
-		<img class="workshop-author_profile" src="https://secure.gravatar.com/avatar/1064bd40c9ade0b3c9cf0e27d9e3432a?s=200&d=mm&r=g" />
+		<img class="workshop-author_profile" src="<?php echo $url; ?>" />
```

Comment:
I would recommend using `get_avatar` here, which will give you the whole `img` tag.